buster/busterbot.py

```diff
@@ -64,11 +64,11 @@ class BusterConfig:
     source: str = ""
 
 
-from buster.documents.base import DocumentsManager
+from buster.retriever import Retriever
 
 
 class Buster:
-    def __init__(self, cfg: BusterConfig, documents: DocumentsManager):
+    def __init__(self, cfg: BusterConfig, documents: Retriever):
```

Comment:
maybe with the renaming we shoudl consider renaming `documents` to something else?

Reply:
you're right, I renamed all instances to retriever